src/EditorFeatures/Test2/Rename/RenameEngineTests.VisualBasicConflicts.vb

```diff
@@ -12,7 +12,7 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.Rename
                 _outputHelper = outputHelper
             End Sub
 
-            <WpfFact(Skip:="798375, 799977")>
+            <WpfFact>
```

Comment:
The `x` in `Console.WriteLine(x)` will change to `y`, which then binds to the local `y` instead of the new parameter `y`. This makes the call to `WriteLine` use the `object`-accepting override instead of the `int`-accepting override.  `D`.